parsec/api/protocol/organization.py

```diff
@@ -12,6 +12,7 @@ class APIV1_OrganizationCreateReqSchema(BaseReqSchema):
 class APIV1_OrganizationCreateRepSchema(BaseRepSchema):
     bootstrap_token = fields.String(required=True)
     expiration_date = fields.DateTime(allow_none=True, required=False)
+    active_users_limit = fields.Integer(allow_none=True, required=True)
```

Comment:
I don't think we should add this here (given how easy it is to call `organization_status` to have this information and all the other configuration)

(of course the same rational can be used against the `expiration_date` field... but this one is already there :trollface:  and we never use it !)